README.md

```diff
@@ -147,6 +147,16 @@ yarn run check:stylelint
 yarn run test
 ```
 
+### Documentation Deployment & CI
+
+CI jobs config scripts can be found under .github/workflows
+
+Docs from `master` branch are hosted on [gh-pages](https://pages.github.com/), and via [Render](https://render.com/) on each open PR.
```

Comment:
I find the wording confusing as it sounds like GitHub Pages hosts PR environments

```suggestion
Every PR is hosted on [Render](https://render.com) with a link on the pull request conversation. Docs from `master` are additionally hosted on [GitHub Pages](https://pages.github.com/).
```